### PR TITLE
Chore/gha trigger

### LINKED ISSUE #38: Trigger GitHub Action build on pull request
Currently, the GitHub action build runs triggered by the push only.
As a reviewer, I want to make sure the PR is building successfully by GH Actions prior reviewing.

### PR DESCRIPTION
**Pull request by zambrovski**
Fix #38

### CODE REVIEW

**Comment by zambrovski** [comment]
Since @smcvb merged the changes of dependbot, there are som conflicts. I believe, this PR must wait until #37 is merged.

**Review by lfgcampos — APPROVED** [review]
Good one.

**Review by sandjelkovic — APPROVED** [review]
LGTM

**Pull request merged**
